pkg/services/business_case.go

```diff
@@ -241,8 +241,10 @@ func NewUpdateBusinessCase(
 		// Once CEDAR endpoint exists, we should be doing validations and submissions in the CEDAR package
 		if businessCase.Status == models.BusinessCaseStatusSUBMITTED &&
 			existingBusinessCase.Status == models.BusinessCaseStatusDRAFT {
-			// Set submitted at time before validations as it is one of the fields that is validated
-			businessCase.SubmittedAt = businessCase.UpdatedAt
+			// Set submitted at times before validations as it is one of the fields that is validated
+			businessCase.SubmittedAt = &updatedAt
+			businessCase.InitialSubmittedAt = &updatedAt
```

Comment:
Do we have a future story to handle the edge case of only writing this field on initial submit? As the code is written, it looks like we'll be over-writing `initialSubmittedAt` every time, right?

Reply:
we don't have a story for this yet...